.azure-pipelines/master.yml

```diff
@@ -30,8 +30,14 @@ stages:
              find . -not -path "*/\.*" -type f -name "*.ipynb"\
               | xargs jupyter nbconvert --to python --output-dir tests\
               --RegexRemovePreprocessor.patterns="['^# exclude-from-export']"
-             python tests/bragg-edge-imaging-2D.py
-             python tests/sans-reduction.py
+             exclude_list=()
```

Comment:
Alternative would be not to nbconvert them into /tests in the first place that might be preferable but I don't have a strong opinion on that.